packages/palette/src/elements/Pagination/SmallPagination.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+import styled from "styled-components"
+import { color } from "../../helpers/color"
+import { space } from "../../helpers/space"
+import { ChevronIcon } from "../../svgs/ChevronIcon"
+import { Flex } from "../Flex"
+
+import { Props } from "./LargePagination"
+
+/** SmallPagination */
+export const SmallPagination = (props: Props) => {
+  const {
+    pageCursors: { previous },
+    onClick,
+    onNext,
+    hasNextPage,
+  } = props
+
+  const handlePrevClick = () => {
+    if (previous) {
+      onClick(previous.cursor, previous.page)
+    }
+  }
+
+  const handleNextClick = () => {
+    onNext()
+  }
+
+  return (
+    <Flex flexDirection="row" width="100%">
+      <PrevButton enabled={previous} onClick={handlePrevClick} />
+      <NextButton enabled={hasNextPage} onClick={handleNextClick} />
+    </Flex>
+  )
+}
+
+const PrevButton = ({ enabled, onClick }) => {
```

Comment:
Can we add types here?

Reply:
Yeah good call. I didn't initially deal with this because what I found wasn't well typed, but no reason to not do it! Took a stab at this with b4a54b8.